cw-orch-daemon/src/sender.rs

```diff
@@ -48,14 +48,29 @@ pub type Wallet = Rc<Sender<All>>;
 
 /// Signer of the transactions and helper for address derivation
 /// This is the main interface for simulating and signing transactions
+#[derive(Clone)]
 pub struct Sender<C: Signing + Context> {
     pub private_key: PrivateKey,
     pub secp: Secp256k1<C>,
     pub(crate) daemon_state: Rc<DaemonState>,
+    pub(crate) options: SenderOptions,
+}
+
+#[derive(Default, Clone)]
+#[non_exhaustive]
+pub struct SenderOptions {
+    pub authz_granter: Option<String>,
```

Comment:
It doesn't have a constructor method, but marked with `non_exhaustive`, which makes impossible creation of desired struct
Using this workaround rn:
```rust
let sender_options = {
    let mut sender_options = SenderOptions::default();
    sender_options.authz_granter = Some(granter);
    sender_options
};
```